charts/restate-helm/templates/service.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Should we set the `namespace` for the service `{{ include "restate.fullname" . }}` below as well?